types/node/process.d.ts

```diff
@@ -110,6 +110,14 @@ declare module 'process' {
             }
             // Alias for compatibility
             interface ProcessEnv extends Dict<string> {
+                /**
+                 * Package name (from package.json)
+                 */
+                npm_package_name: string;
```

Comment:
These are only set when executed via `npm run` etc, I don't think we should add these, the index signature is sufficient.